src/eva/data/mon_data_space.py

```diff
@@ -291,7 +292,8 @@ def read_ieee(self, file_name, coords, dims, ndims_used, dims_arr, nvars, vars,
 
         # find cycle time in filename and create cycle_tm as datetime object
         cycle_tm = None
-        cycstrs = file_name.split('.')
+        fname = file_name.replace('/', '.')
+        cycstrs = fname.split('.')
```

Comment:
If you don't need the variable `fname` anywhere else, you can combine these two lines into:
`cycstrs = file_name.replace('/', '.').split('.')`

However, it doesn't need to be changed.

Reply:
No problem.  Change made.